cmd/proplet/main.go

```diff
@@ -59,9 +61,13 @@ func main() {
 
 		return
 	}
-	wazero := proplet.NewWazeroRuntime(logger, mqttPubSub, cfg.ChannelID)
 
-	service, err := proplet.NewService(ctx, cfg.ChannelID, cfg.ThingID, cfg.ThingKey, cfg.LivelinessInterval, mqttPubSub, logger, wazero)
+	runtime := runtimes.NewWazeroRuntime(logger, mqttPubSub, cfg.ChannelID)
```

Comment:
I prefer doing it with `if-else`, then re-assigning and calling New two times in the worst case. It documents better the intention.